src/main/java/com/cleanroommc/groovyscript/compat/mods/thaumcraft/arcane/ArcaneRecipeBuilder.java

```diff
@@ -23,6 +25,18 @@ public interface ArcaneRecipeBuilder {
     @RecipeBuilderMethodDescription(field = "aspects")
     ArcaneRecipeBuilder aspect(AspectStack aspect);
 
+    @RecipeBuilderMethodDescription(field = "aspects")
+    default ArcaneRecipeBuilder aspect(AspectStack... aspect) {
+        for (var entry : aspect) aspect(entry);
+        return this;
+    }
+
+    @RecipeBuilderMethodDescription(field = "aspects")
+    default ArcaneRecipeBuilder aspect(Collection<AspectStack> aspect) {
+        for (var entry : aspect) aspect(entry);
+        return this;
+    }
```

Comment:
`aspects` rather than `aspect`?

Reply:
That would be inconsistent with the rest of grs

Reply:
SAD!